.github/workflows/auto_docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Automatic Documentation for Launcher
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Should this run just when the files with the parameters are changed? E.g. something like https://github.com/huggingface/text-generation-inference/blob/main/.github/workflows/build_documentation.yml#L5C5-L6C25

Reply:
Hmm, does it make sense run this everytime there's a change in `main.rs` instead, given it's where the docs come from?

Reply:
fixed this.

Reply:
Yes great idea!